carnival.cabal

```diff
@@ -137,9 +139,6 @@ library
                  , yesod-text-markdown
 
 executable         carnival
-    if flag(library-only)
```

Comment:
It's slower to build, it increases the size of the slug, and it complicates the cabal file.

I can do a version which uses a separate executable to see what the actual differences are.